pkg/minishift/registration/utils.go

```diff
@@ -35,21 +34,24 @@ type RegistrationParameters struct {
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 	commander := provision.GenericSSHCommander{Driver: host.Driver}
-	registrator, err := DetectRegistrator(commander)
-	if err == ErrDetectionFailed {
+	registrator, supportRegistration, err := DetectRegistrator(commander)
+	if ! supportRegistration {
 		fmt.Println("Distribution doesn't support registration")
```

Comment:
I think this should be debug at best. In the default case with the b2d or CentOS image, there won't be registration, still the user will see this everytime he stops the machine. That might throw him off.